x-pack/test/login_selector_api_integration/apis/login_selector.ts

```diff
@@ -124,6 +124,57 @@ export default function ({ getService }: FtrProviderContext) {
         }
       });
 
+      it('should redirect to URL from relay state in case of IdP initiated login only for providers that explicitly enabled that behaviour', async () => {
```

Comment:
__note:__ it's in a Login Selector test suite since it's the only suite that uses new config format.